athena/tests/integration_tests/eunomia/eval/models/entities/evaluation_runner.py

```
@@ -0,0 +1,191 @@
+        # Inject the global cost calculator
+        scenario.llm_cost_calculator = self.llm_cost_calculator
+        scenario.run_name = self.run_name
+
+        return scenario
+
+    def _load_manifest(self, manifest_path: Path) -> dict:
+        with manifest_path.open("r", encoding="utf-8") as f:
+            manifest = yaml.safe_load(f)
+        return manifest
+
+    def _create_exercise_from_manifest(self, manifest: dict, scenario_dir: Path):
```

Pylint (3.3.7)

[refactor] Too many local variables (33/15)

(R0914)